CMakeLists.txt

```
@@ -2860,6 +2860,55 @@ if (SDL_DIALOG)
   endif()
 endif()
 
+sdl_sources("${SDL3_SOURCE_DIR}/src/process/SDL_process.c")
+if(WINDOWS)
+  sdl_glob_sources("${SDL3_SOURCE_DIR}/src/process/windows/*.c")
+  set(SDL_PROCESS_WINDOWS 1)
+  set(HAVE_SDL_PROCESS TRUE)
+else()
+  check_c_source_compiles("
+#include <spawn.h>
+#include <unistd.h>
+
+int main(void)
+{
+    int pipes[2];
+    int pid;
+
+    const char * args[] = {
+      \"/usr/nonexistence\",
```

Might as well be

```suggestion
      \"/bin/false\",
```

which is something that really exists.

---

For reference, this is just to check if the code compiles at all, I don't think it even needs a list of strings for arguments at all.